build-compose.sh

```diff
@@ -18,8 +18,9 @@ then
   fi
 fi
 
+if [[ ! $(docker images -q tsuru-build) ]]; then docker build -t tsuru-build -f Dockerfile.build .; fi;
```

Comment:
What do you think about always running something like:
```
context=$(mkdir -d)
docker build -t tsuru-build -f Dockerfile.build $context
rmdir $context
```

This would ensure the build runs almost instantaneously if the image is already up to date but it would also rebuild the image if changes to Dockerfile.build were detected, instead of requiring the tsuru-build to be removed manually.

Reply:
Good idea.